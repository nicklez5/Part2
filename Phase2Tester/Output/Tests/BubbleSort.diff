--- SelfTestCases/BubbleSort.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/BubbleSort.out	2020-02-17 22:35:01.661459697 -0800
@@ -1,22 +1,5 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+--- Error (ProgramFault) ---
+expecting argument 2 to be an integer, found undefined value
+  Main() line 15
+  BBS.Start(<Heap.0>+0 10) line 26
+  BBS.Print(<Heap.0>+0) line 113
