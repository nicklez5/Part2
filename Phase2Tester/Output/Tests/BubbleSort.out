--- Error (ProgramFault) ---
expecting argument 2 to be an integer, found undefined value
  Main() line 15
  BBS.Start(<Heap.0>+0 10) line 26
  BBS.Print(<Heap.0>+0) line 113
